stream.js

```diff
@@ -183,6 +183,16 @@ Stream.prototype.liveResume = function liveResume() {
   this.resume()
 }
 
+Stream.prototype.postCompactionReset = function postCompactionReset(offset) {
+  if (!this.live) throw new Error('postCompactionReset only works with live')
+  if (this.max || this.max_inclusive) {
+    throw new Error('postCompactionReset only works with no upper bound')
```

Comment:
In index.js i mean

Reply:
Ah, I see. I think in any case it might blow up in the user interface (maybe app crash) but it might be a bit friendlier to do it before compaction, indeed.